examples/config-many-files/teams-config.yml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: v1
+kind: options
+spec:
+  config:
+    team_name: default
+    options:
+      distributed_interval: 3
+      distributed_tls_max_attempts: 3
+      logger_plugin: tls
+      logger_tls_endpoint: /api/v1/osquery/log
+      logger_tls_period: 10
+    decorators:
+      load:
+        - "SELECT version FROM osquery_info"
+        - "SELECT uuid AS host_uuid FROM system_info"
+      always:
+        - "SELECT user AS username FROM logged_in_users WHERE user <> '' ORDER BY time LIMIT 1"
+      interval:
+        3600: "SELECT total_seconds AS uptime FROM uptime"
+---
+apiVersion: v1
+kind: options
+spec:
+  config:
+    team_name: workstations
+    options:
+      distributed_interval: 3
+      distributed_tls_max_attempts: 3
+      logger_plugin: tls
+      logger_tls_endpoint: /api/v1/osquery/log
+      logger_tls_period: 10
+    decorators:
+      load:
+        - "SELECT version FROM osquery_info"
+        - "SELECT uuid AS host_uuid FROM system_info"
+      always:
+        - "SELECT user AS username FROM logged_in_users WHERE user <> '' ORDER BY time LIMIT 1"
+      interval:
+        3600: "SELECT total_seconds AS uptime FROM uptime"
```

Comment:
I... clearly haven't used `interval` decorators or I probably would have noticed this in the **existing** fleet config.

It looks like `interval` is a map of `int` (seconds) to `string` (query). However, what happens if I want to run two queries at the same interval? I suspect maybe this should instead be something like:

```
interval:
  - query: "SELECT thing FROM some_place"
    interval: 3600
  - query: "SELECT other_thing FROM other_place"
    interval: 3600
```